src/database/run-seed.ts

```diff
@@ -0,0 +1,44 @@
+import { DataSource } from 'typeorm';
+import { AccommodationCategorySeeder } from './seeders/accommodation-category.seeder';
+import { AccommodationAmenitySeeder } from './seeders/accommodation-amenity.seeder';
+import { Logger } from '@nestjs/common';
+import { dataSourceConfig } from 'src/shared/configs/data-source.config';
+import {
+  Accommodation,
+  AccommodationCategory,
+  Amenity,
+} from 'src/modules/accommodations/entities';
+import { User } from 'src/modules/users/entities/user.entity';
+
+export class MainSeeder {
```

Comment:
Can we also implement the Seeder interface form typeorm here as well?

Reply:
Sure. Done